PresentScreenings/AppDelegate.cs

```diff
@@ -86,8 +86,8 @@ void ShowSaveAs(NSObject sender)
                 FilmInfo.SaveFilmInfoAsXml(ScreeningsPlan.FilmInfos, Path.Combine(directory, "filminfo.xml"));
 
                 // Write film ratings.
-                string filmFanRatingsPath = Path.Combine(directory, "ratings.csv");
-                new ListWriter<FilmFanFilmRating>(filmFanRatingsPath, FilmFanFilmRating.WriteHeader).WriteListToFile(ScreeningsPlan.FilmFanFilmRatings);
+                string ratingsPath = Path.Combine(directory, "ratings.csv");
+                new ListWriter<FilmFanFilmRating>(ratingsPath, FilmFanFilmRating.WriteHeader).WriteListToFile(ScreeningsPlan.FilmFanFilmRatings);
```

Comment:
Hm. Well, let's agree on that for the time being.
We will find a better name for `FilmFanFilmRating` in the end.
What about `FilmRating`?

Reply:
We already have a class `FilmRating`, handling film rating values.
Class `FilmFanFilmRating` links ratings to films and individuals.
But, in the end, we will find the best names.

Reply:
We already have a class `FilmRating`, handling film rating values.
Class `FilmFanFilmRating` links ratings to films and individuals.
But, in the end, we will find the best names.